src/lib/modules/ping/ping_test.go

```diff
@@ -25,4 +27,15 @@ func TestHandlerPing(t *testing.T) {
 			log.Println(*a)
 		}
 	}
-}
\ No newline at end of file
+}
+
+func TestHandlerFloodPing(t *testing.T) {
+	for _, inst := range urls {
+		a := HandleFloodPing(&inst, 1000)
+		if reflect.TypeOf(a).String() != "uint64" {
```

Comment:
please make the error condition wider. like, after implementing min/avg/max/mdev in the flood ping, it would be good to check if the returned value is abnormal or not.

Reply:
What would be abnormal in this case?

Reply:
might be like a ping being 0ms, ping above 1000ms, or some negative integer or a string being passed on due to wrong scrapping. This would be necessarily helpful if we have some contributor which would be involved in improving this module and has by mistake messed up with the scrapping algo. 

What do you think? @ankitjena

Reply:
Yes, that sounds good.